cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -68,7 +68,6 @@ import (
 	"k8s.io/kubernetes/pkg/features"
 	"k8s.io/kubernetes/pkg/serviceaccount"
 	"k8s.io/kubernetes/pkg/util/configz"
-	utilflag "k8s.io/kubernetes/pkg/util/flag"
```

Comment:
there is another commit with the same name. One is wrong.

Reply:
@soltysh Given that you're the author of both patches with `allow running bare kube-controller-manager` in the name, can you provide guidance as to which should be retained or if they should be squashed?

Reply:
Squashed.